### PR TITLE
fix: add timeout options when exec arp

### PR DESCRIPTION
**Pull request by howel52**
https://github.com/DylanPiercey/local-devices/issues/12

### CODE REVIEW

File: __tests__/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const find = require('../src/index')
 var cp = require('mz/child_process')
 
 const TEN_MEGA_BYTE = 1024 * 1024 * 10
+const MAX_ARP_TIME = 60 * 60 * 10
```
Comment (natterstefan) [inline __tests__/index.js]:
```suggestion
const TIMEOUT_MS = 60 * 1000
```

I would suggest renaming the variable and increase the timeout to 60s or decrease to 30s. And update in the other files as well, please.

Comment (howel52) [reply __tests__/index.js]: Totally agree 👍

**Comment by howel52** [comment]
> Thank you for your PR and help @howel52. I have just one thing I'd suggest to change.
> 
> @DylanPiercey Do you agree with the value for the timeout? _Edit_: and we might need to think about a way developers can define the timeout and maxBuffer themselves. But that's most likely out of scope of this PR.

yeah,  I think developers define the options by themselves is right. 

And now, In this PR, I have modified the code for suggestions on code review.   @natterstefan

**Comment by DylanPiercey** [comment]
@howel52 thanks for this PR, it looks great 🙂.

I agree with @natterstefan that this should be configurable, but it isn’t absolutely required that we do that in this PR.

This change will of course require a major release since the current default timeout is effectively `Infinity`. We can then do a minor release to expose both the max buffer and timeout.

File: __tests__/index.js (hunk 1 above)
Comment (DylanPiercey) [reply __tests__/index.js]: This is not a big deal, but I’d personally like to see the variable called `ONE_MINUTE`, to make it clear what the intent is without having to do the (fairly simple) calculation yourself. It also better aligns with the `TEN_MEGA_BYTE` const we already have.

Comment (howel52) [reply __tests__/index.js]: I have changed the variable name

**Comment by howel52** [comment]
I want to know when this PR can be merged. And when will the new npm package be released ?

**Comment by natterstefan** [comment]
Hi @howel52.

I have just merged it, but it's up to @DylanPiercey to release a new version of the package.

@DylanPiercey we could then probably also include (merge+release) this PR: https://github.com/DylanPiercey/local-devices/pull/9.

**Comment by howel52** [comment]
> Hi @howel52.
> 
> I have just merged it, but it's up to @DylanPiercey to release a new version of the package.
> 
> @DylanPiercey we could then probably also include (merge+release) this PR: #9.

thx :)

**Pull request merged**